File: src/test/fixtures/sample.py
from flask import Flask, request
from sqlalchemy import Column, Integer, String
import os

app = Flask(__name__)

class UserModel:
    """SQLAlchemy User model"""
    id = Column(Integer, primary_key=True)
    name = Column(String(100))
    email = Column(String(200))

    def get_display_name(self):
        return self.name

@app.route('/api/users', methods=['GET'])
def list_users():
    """Returns all users"""
    return []

@app.route('/api/users', methods=['POST'])
def create_user():
    """Creates a new user"""
    return {}, 201

async def background_job(task_id: str):
    pass

File: src/test/fixtures/sample.ts
import { Request, Response } from 'express';
import { Router } from 'express';

export interface UserDto {
  id: number;
  name: string;
  email: string;
}

/**
 * Handles user-related HTTP operations
 */
export class UserController {
  private router: Router;
  private baseUrl: string;

  constructor() {
    this.router = Router();
    this.baseUrl = '/api/users';
  }

  /**
   * Returns a list of all users
   */
  async getUsers(req: Request, res: Response): Promise<void> {
    res.json([]);
  }

  async createUser(req: Request, res: Response): Promise<void> {
    res.status(201).json({});
  }
}

/**
 * Standalone utility — formats a user display name
 */
export async function formatUserName(user: UserDto): Promise<string> {
  return `${user.name} <${user.email}>`;
}

export function getUserById(id: number): UserDto | null {
  return null;
}
